client/components/jetpack/card/jetpack-product-card/features-item.tsx

```diff
@@ -9,24 +9,37 @@ import React, { FunctionComponent } from 'react';
 import Gridicon from 'components/gridicon';
 import InfoPopover from 'components/info-popover';
 import { preventWidows } from 'lib/formatting';
-import type { FeaturesItem } from './types';
+
+/**
+ * Type dependencies
+ */
+import type { ProductCardFeaturesItem } from './types';
 
 export type Props = {
-	item: FeaturesItem;
+	item: ProductCardFeaturesItem;
 };
 
 const DEFAULT_ICON = 'checkmark';
 
 const JetpackProductCardFeaturesItem: FunctionComponent< Props > = ( { item } ) => {
-	const { icon, text, description } = item;
+	const { icon, text, description, subitems } = item;
 
 	return (
 		<li className="jetpack-product-card__features-item">
-			<div className="jetpack-product-card__features-summary">
-				<Gridicon className="jetpack-product-card__features-icon" icon={ icon || DEFAULT_ICON } />
-				<p className="jetpack-product-card__features-text">{ preventWidows( text ) }</p>
+			<div className="jetpack-product-card__features-main">
+				<div className="jetpack-product-card__features-summary">
+					<Gridicon className="jetpack-product-card__features-icon" icon={ icon || DEFAULT_ICON } />
+					<p className="jetpack-product-card__features-text">{ preventWidows( text ) }</p>
+				</div>
+				{ description && <InfoPopover>{ preventWidows( description ) }</InfoPopover> }
 			</div>
-			{ description && <InfoPopover>{ preventWidows( description ) }</InfoPopover> }
+			{ subitems && subitems?.length > 0 && (
```

Comment:
I think `subitems && subitems?.length` can be simplified to `subitems?.length`.

Reply:
I thought it would, be TS is oddly still showing a "might be undefined" type of error. I'll see if my version is the latest.

Reply:
Oh, interesting. Well if TS doesn't like it, I guess we should leave it as it is now.

Reply:
Yeah, TS doesn't handle that language construct well yet.